db/dcrpg/upgrades.go

```diff
@@ -486,6 +516,48 @@ func (pgb *ChainDB) handleVinsTableMainchainupgrade() (int64, error) {
 	return rowsUpdated, nil
 }
 
+func (pgb *ChainDB) handleTicketsBlockTimeUpgrade() (int64, error) {
+	log.Info(" - Retrieving all the ticket transactions with blocktime")
+	ticketsBlockTime, err := retrieveTicketsTxsBlockTime(pgb.db)
+	if err != nil {
+		return 0, fmt.Errorf("Unable to retreive all a tickets block time: %v", err)
```

Comment:
`unable to retrieve ticket block times: %v` (The string in an `error` type should start lower case).

Reply:
ok. Noted